src/core/server/plugins/plugins_system.ts

```diff
@@ -40,7 +42,7 @@ export class PluginsSystem {
   public async startPlugins(deps: PluginsServiceStartDeps) {
     const exposedValues = new Map<PluginName, unknown>();
     if (this.plugins.size === 0) {
-      return exposedValues;
+      return new Map();
```

Comment:
just curious - what was affected ?

Reply:
Ah I can put this back, this was a vestige from when I had changed this method more materially. Thanks for catching.